db/migrate/20190912143132_add_release_field_to_os.rb

```diff
@@ -0,0 +1,40 @@
+class AddReleaseFieldToOs < ActiveRecord::Migration[5.2]
+  class Operatingsystem < ApplicationRecord
+    self.inheritance_column = nil
+  end
+
+  def up
+    add_column :operatingsystems, :release, :string, :limit => 16, :default => "", :null => false
+
+    ok = true
+    Operatingsystem.all.each do |os|
+      if os.minor&.include?('.')
+        minor_release_array = os.minor.split('.')
+        minor = minor_release_array[0]
+        release = minor_release_array[1]
```

Comment:
I believe there can be more elements in the array, if we have minor with more parts i.e. `"4.33.2345"` so we should use `[1..-1]` to take all what's left in the array and join it back together.